docker-compose.meltano.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.3'
```

Comment:
Do we still need `docker-compose.meltano.yml` if we are also standing up meltano in `docker-compose.yml`?

Reply:
No, I'll remove it.